src/subscription/__tests__/subscribe-test.js

```diff
@@ -125,13 +122,21 @@ describe('Subscribe', () => {
 
     // GraphQL `subscribe` has the same call signature as `execute`, but returns
     // AsyncIterator instead of Promise.
-    const subscription = subscribe(
-      emailSchema,
-      ast,
-      data,
-      null, // context
-      { priority: 1 }
-    );
+    return {
+      subscription: subscribe(
+        emailSchema,
+        ast,
+        data,
+        null, // context
+        { priority: 1 }
+      ),
+      sendImportantEmail,
```

Comment:
small thing, but moving this to the first key in this return object would make it more obvious that more than one thing are being returned, otherwise it's easy to mistake this as another argument to subscribe